ROX-25216: S3 Compatible Backup Integration (#78737)
openshift/openshift-docs: modules/perform-on-demand-backups-s3-api-compatible-services.adoc

// Module included in the following assemblies:
//
// * integration/integrate-with-s3-api-compatible-services.adoc
:_mod-docs-content-type: PROCEDURE
[id="perform-on-demand-backups-s3-api-compatible-services_{context}"]
= Performing on-demand backups on an S3 API compatible bucket

Use the {product-title-short} portal to trigger manual backups of {product-title} to an S3 API compatible bucket.

.Prerequisites

> * You must have already integrated {product-title} with an S3 API compatible bucket.

Reviewer: ```suggestion
* You have integrated {product-title-short} with an S3 API compatible bucket.
```

Reviewer: A suggestion - elsewhere you refer to the S3 API compatible service, not bucket. I was interpreting this as the service was something that you configured the integration with, and then the bucket is the storage device that you actually backed up to. So I think here you might want to use "you have integrated RHACS with an S3 API compatible service"?

Author: Yeah `S3 API compatible service` makes more sense here. I'll change it.

.Procedure
. In the {product-title-short} portal, go to *Platform Configuration* -> *Integrations*.
. Under the *External backups* section, click *S3 API Compatible*.
. Select the integration name for the S3 bucket where you want to do a backup.

> . Click *Trigger Backup*.

Reviewer: Is backup capitalized? (I think Patternfly style has sentence case so it shouldn't be, I think?)

Author: You are right, backup is not capitalized. I'll change it.

[NOTE]
====

> When you select the *Trigger Backup* option, there is no notification.

Reviewer: ```suggestion
When you select *Trigger Backup*, there is no notification.
```

> However, {product-title} begins the backup task in the background.

Reviewer: ```suggestion
However, {product-title-short} begins the backup task in the background.
```
